packages/parser/lib/css_lexer.re

```diff
@@ -549,9 +549,9 @@ let rec get_next_token = buf => {
   | greater_operator
   | less_operator
   | equal_sign
-  | only_operator // TODO: maybe IDENT
-  // | or_operator // TODO: maybe IDENT
-  | and_operator // TODO: maybe IDENT
+  | only_operator
+  // | or_operator
```

Comment:
This must be a todo; when called, it conflicts with some other `or` which I can't trace atm